README.md

```diff
@@ -39,10 +52,6 @@ A comprehensive experience when using [Facebook's Jest](https://github.com/faceb
 
 <img src="https://github.com/jest-community/vscode-jest/raw/master/images/vscode-jest.gif" alt="Screenshot of the tool" width="100%">
 
-## Maintainers
-
-Orta Therox ([@orta](https://github.com/orta)), Vincent Voyer ([@vvo](https://github.com/vvo)) & ConnectDotz ([@connectdotz](https://github.com/connectdotz)).
```

Comment:
IMO, you should switch this to you 👍🏻

Reply:
I just happened to have some freecycle recently so decided to take on these big-ticket items. Unfortunately, I didn't set a good example for the more "progressive" PR, but I think this kind of change is probably not a good fit for that. Nevertheless, I probably should have open a discussion thread before pulling the trigger... Please do feel free to voice your opinions, PR is not merged in, it's not too late. 

Ok, now to the actual point, I think we need more maintainers, not less, so we can have more people bouncing ideas and widening perspectives. How do you decide the maintainers? Do you have any suggestions on who else we can add? I know @stephtr is busy recently, but he has been very helpful in our recent PRs, do you think we could add him as the maintainer?

> IMO, you should switch this to you 👍🏻

You mentioned this couple of times, I might change my mind in the future, but right now I quite enjoy you being the owner so I have somebody to discuss and learn from 😄